src/components/TextArea/__snapshots__/TextArea.test.tsx.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TextArea component compontent snapshots default 1`] = `
+<div>
+  <div
+    class="usa-form-group"
+  >
+    <label
+      class="usa-label"
+      for="input-type-textarea"
+    >
+      Testing TextArea
+    </label>
+    <textarea
+      aria-describedby=""
+      class="usa-textarea"
+      id="input-type-textarea"
+      name="testing-textarea"
+    />
+  </div>
+</div>
+`;
+
+exports[`TextArea component compontent snapshots required and error 1`] = `
+<div>
+  <div
+    class="usa-form-group"
+  >
+    <label
+      class="usa-label"
+      for="input-type-textarea"
+    >
+      Testing TextArea
+      *
+    </label>
+    <textarea
+      aria-describedby=""
+      class="usa-textarea"
+      id="input-type-textarea"
+      name="testing-textarea"
+      placeholder="Can you place this?"
+      required=""
```

Comment:
I was noticing this on my components, booleans were rendering as strings, was wondering if you've encountered this before